src/joint.dia.link.js

```diff
@@ -1446,7 +1436,7 @@ joint.dia.LinkView = joint.dia.CellView.extend({
         // if are simulating pointerdown on a link during a magnet click, skip link interactions
         if (evt.target.getAttribute('magnet') != null) return;
 
-        var interactive = _.isFunction(this.options.interactive) ? this.options.interactive(this, 'pointerdown') : this.options.interactive;
+        var interactive = _.isFunction(this.options.interactive) ? this.options.interactive(this) : this.options.interactive;
```

Comment:
First of all I think we should avoid using this construct `var interactive = _.isFunction(this.options.interactive) ? this.options.interactive(this) : this.options.interactive;` on multiple places. As It is already implemented in the `CellView.prototype.can()` method.

When you look at the `pointerdown` method, it stops executing the method rigth at the beginning when `interactive=false`. If there was not such a check the method would carry on executing and terminates with no effect at some other place where `can('feature')` would return `false` (because can(feature) always return `false` if `interactive=false`).